datacenter/ucp/2.2/guides/access-control/deploy-stack-to-collection.md

```diff
@@ -0,0 +1,95 @@
+---
+title: Deploy stack resources to a collection 
+description: Learn how to manage user access to stack resources by using collections. 
+keywords: UCP, authentication, user management, stack, collection, role
+---
+
+A stack file is the definition of an application, and in the stack file,
+you can apply access labels to specify access to application resources.
+UCP interprets an access label to be the name of a collection that users are
+granted access to by an administrator.
+[Learn about managing access to resources by using collections](manage-access-with-collections.md).  
+
+Specify a collection for a resource by adding a `labels:` dictionary to the
+stack file and assigning the `com.docker.ucp.access.label` key. For example,
+a stack file that puts a resource in the `database` collection might look
+similar to this:  
+
+```none
+services:
+   db:
+     image: mysql:5.7
+     deploy:
+      labels:
+        com.docker.ucp.access.label: database
```

Comment:
This example uses a legacy label, but new users will never be exposed to this UX. We should instead document the "full path" format with something such as `com.docker.ucp.access.label: /Shared/database`